lib/package-managers/npm.js

```diff
@@ -56,25 +58,26 @@ function view(packageName, field, currentVersion) {
     npmConfig['full-metadata'] = field === 'time';
 
     return pacote.packument(packageName, npmConfig).then(result => {
-        if (field.startsWith('dist-tags.')) {
-            const [tagName, version] = field.split('.');
-            if (result[tagName]) {
-                return result[tagName][version];
-            }
-        } else if (field === 'versions') {
-            return Object.keys(result[field]);
+        if (field === 'versions') {
+            return result.versions;
         } else {
-            return result[field];
+            return {
+                [field]: result[field],
+                versions: result.versions,
+            };
```

Comment:
This is problematic since it changes the semantics of the `view` function in an unconventional way. Previously `view` returned the value of the field requested, which was simple and intuitive. Always returning `versions` is counterintuitive, even though it accomplishes what you want. i.e. It's a bit too much of a hack.

I would recommend the following approach: Define two functions: `viewOne` and `viewMany`. `viewMany` will be the same as the existing `view` but take an array of fields as the second argument and return an object containing each of those fields with their values. This is a more general solution than the one presented. Then `viewOne` simply wraps `viewMany` and returns only the field value, preserving the current behavior (and requiring no change on the caller side).